Ficha9/grafo.c

```diff
@@ -106,8 +191,16 @@ vetor* grafo_arestasEntrada(grafo* g, int i)
 int grafo_completo(grafo* g)
 {
     /* alinea 1.4 */
+    if(g == NULL){
+        return -1;
+    }
 
-    return 0;
+    for(int i=0;i<g->tamanho;i++){

```

Comment:
Can you estimate the complexity of this algortihm? In O notation?

Reply:
O(n) or O(g->tamanho)?

Reply:
Only if every function call was O(1), what is the complexity of `grafo_arestasSaida`

Reply:
Hmmm....okok. So, since the function 'grafo_arestasSaida' has a complexity of O(g->tamanho), that means the function 'grafo_completo' has a complexity of O(g->tamanho²) right?.

Reply:
exactly, the function is O(N square).

I believe it can be done in O(N)